### PR TITLE
Display last updated time for VM

### PR DESCRIPTION
**Pull request by ravening**
### Description
Add a field in api response and ui to display last updated time

```
(local) mgt01 > list virtualmachinesmetrics filter=lastupdated,
{
  "count": 1,
  "virtualmachine": [
    {
      "lastupdated": "2021-02-10T15:43:14+0000"
    }
  ]
}
(local) mgt01 > list virtualmachines filter=lastupdated,
{
  "count": 1,
  "virtualmachine": [
    {
      "lastupdated": "2021-02-10T15:43:14+0000"
    }
  ]
}
```

Screenshot 2021-02-11 at 15 08 59

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@ravening conflict here

**Comment by shwstppr** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
LGTM. tested with simulator

**Comment by rhtyd** [comment]
@ravening pl fix the conflict

**Comment by sureshanaparti** [comment]
ping @ravening can you resolve the conflicts

**Comment by ravening** [comment]
> @ravening pl fix the conflict

@rhtyd @sureshanaparti fixed it

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by RodrigoDLopez — CHANGES_REQUESTED** [review]
Minor change request.

**Review by rhtyd — APPROVED** [review]

File: api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java [hunk 1]
```diff
@@ -82,6 +82,10 @@
     @Param(description = "the date when this virtual machine was created")
     private Date created;
 
+    @SerializedName("lastupdated")
+    @Param(description="the date when this virtual machine was updated last time")
```
[the remark this answers is not in the document]
Comment (rhtyd) [reply api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java]: I've added and checked.

**Comment by rhtyd** [comment]
Tests LGTM, failures not related to this PR.

**Pull request merged**